INTEGRATION.md

```diff
@@ -0,0 +1,71 @@
+## Add in package.json for auto download of vscode-commons
+
+- Dependencies to other extensions. The identifier of an extension is always ${publisher}.${name}. For example: vscode.csharp.
+
+```
+  "extensionDependencies": ["redhat.vscode-commons"],
+```
+
+## Add the bellow code in your client extensions
+
+```
+interface TelemetryEvent {
+  extensionName: string; // name of extension
+  uuid?: string;
+  type?: string; // type of telemetry event such as : identify, track, page, etc.
+  name?: string;
+  properties?: any;
+  measures?: any;
+}
+
+async function telemetry(context: vscode.ExtensionContext) {
+  /*
+    Set segment_key in context.globalState
+    provide your custom segment key or comment below two lines
+  */
+  const SEGMENT_KEY = "abcdefghijklmnopqrstuvwxyz";
+  context.globalState.update("SEGMENT_KEY", SEGMENT_KEY);
+
+  // To get an instance of the Extension
+  const vscodeCommons = vscode.extensions.getExtension("redhat.vscode-commons");
```

Comment:
@fbricon 
Will this not help in avoiding static calls? 
in-ref : https://github.com/redhat-developer/vscode-commons/pull/3#discussion_r530894370